libs/infra-metrics/src/lib/dogStatsD.ts

```diff
@@ -1,4 +1,8 @@
-import { StatsD, ClientOptions } from 'hot-shots'
+import { StatsD, ClientOptions, Tags } from 'hot-shots'
+
+export const METRICS_PROVIDER = 'Metrics'
```

Comment:
Is this the correct place for this?

Reply:
Not really, since it is not provided by infra-metrics and this library is not (necessarily) a nest library.

You can move this to your other module or instantiate DogStatsD manually. But it would be awesome to create a new metric nest module which can be used like this:

```
import { MetricsModule, MetricsService } from '@island.is/nest/metrics'

@Module({
  imports: [
    MetricsModule({ prefix: 'license-service' }),
  ],
})
class MyModule {}

// elsewhere
class MyService {
  constructor(private metrics MetricsService) {}

  yo() {
    this.metrics.increment('yo')
  }
}
```

The MetricsService would abstract the StatsD client and just expose what we need.

Reply:
OK, did think about it and it's The Right Thing To Do™️ , I'll add it